support/gathering-cluster-data.adoc

```diff
@@ -38,14 +38,16 @@ include::modules/gathering-data-specific-features.adoc[leveloffset=+2]
 
 ifndef::openshift-rosa,openshift-dedicated[]
 * xref:../nodes/cma/nodes-cma-autoscaling-custom.adoc#nodes-cma-autoscaling-custom-gather[Gathering debugging data] for the Custom Metrics Autoscaler.
+* link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy]
 endif::openshift-rosa,openshift-dedicated[]
 
-* link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy]
+ifdef::openshift-rosa[]
+* xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.html[{product-title} update life cycle]
+endif::openshift-rosa[]
 
-ifdef::openshift-rosa,openshift-dedicated[]
-// Gathering audit logs
-include::modules/gathering-data-audit-logs.adoc[leveloffset=+2]
```

Comment:
This isn't still included after Mike's changes have gone through, but it also doesn't seem to be being included correctly before them - I thought it might be an ifeval being evaluated incorrectly but despite this file existing in this location, this include isn't working and VS Code can't find it to open with ctrl+click. Suspect naming error, looking into it. This isn't an issue with the PR specifically, but the PR does remove this reference to the file. Rewatching recording to see if I'm misremembering something and will come back and delete this comment if so.